index.html

```diff
@@ -201,25 +201,43 @@ <h2>Music</h2>
+                </p>
+                <footer>
+                    <a url="https://www.unesco.de/en/culture-and-nature/intangible-cultural-heritage/demoscene-culture-digital-real-time-animations">UNESCO Intangible Cultural Heritage page on the demoscene</a>.
+                </footer>
+            </blockquote>
+
+            <p>
+                The demoscene is computer art subculture extending over several continents, and which explores artforms on computer platforms.
+                Often following self-imposed sets of rules or constraints, they include graphics, music, game or animation, but the most typical one are the "demos".
+            </p>
+
+            <p>
+                Demos stand at the crossroad of short films, music videos and video games.
+                Although they present a non-interactive experience, often music driven, like a music video does, they are rendered in real-time like video games are.
+                Intended to awe their audience, they are a demonstration of programming, visual art and musical skills.
+            </p>
+
+            <p>
+                64k intros are demos with an added arbitrary limitation on the size: they consist of a single binary file of no more than 65,536 bytes.
+                With no extra assets, no network access and no extra libraries, they intend to further astonish the viewer with what can be achieved within such a small size.
             </p>
 
             <p>
-                64k intros are demos with an added arbitrary limitation on the size: they consist of a single binary file of no more
-                than 65,536 bytes. No extra assets, no network access, no extra libraries: the usual rule
-                is that it should run on a freshly-installed system (often Microsoft Windows) with up-to-date drivers.
+                Like other demoscene productions, 64k intros are usually shown during competitions at festivals known as demoparties.
+                But despite being released online, they rarely get the attention they deserve.
+                This website intends to show some noteworthy examples of the genre, and gather some documentation and resources for creators, newcomers and seasoned alike, who are interested in this kind of size coding.
             </p>
 
             <p>
-                This website intends to show some noteworthy examples of this genre, and gather some documentation and resources for
-                creators, newcomers and seasoned alike, interested in this kind of size coding. You can contribute to this website by
-                opening pull requests on GitHub.
+                You can contribute to this website by <a url="https://github.com/64k-scene/64k-scene.github.io/pulls">opening pull requests on GitHub</a>.
```

Comment:
"a href=..."

Reply:
Fixed in [0845ce5](https://github.com/64k-scene/64k-scene.github.io/pull/2/commits/0845ce5586ce675a9e1a2a6b4f560ad7a6f06e10).